data/types.ts

```diff
@@ -0,0 +1,9 @@
+export interface Resource {
+    title: string,
+    description: string,
+    url: string,
+    type: string,
```

Comment:
Shall we add a more restricted allowed list for `type`?

Reply:
You mean a string union? Yes, that would be nice

Reply:
You're right, I meant a string union.